bigtable/admin.go

```diff
@@ -1441,10 +1445,17 @@ func UpdateInstanceAndSyncClusters(ctx context.Context, iac *InstanceAdminClient
 
 // RestoreTable creates a table from a backup. The table will be created in the same cluster as the backup.
 func (ac *AdminClient) RestoreTable(ctx context.Context, table, cluster, backup string) error {
-	ctx = mergeOutgoingMetadata(ctx, ac.md)
-	prefix := ac.instancePrefix()
-	backupPath := prefix + "/clusters/" + cluster + "/backups/" + backup
+	return ac.RestoreTableTo(ctx, ac.instance, table, cluster, backup)
+}
```

Comment:
Why do you need two methods here? You probably can do this with `if diffInstance != nil`

Reply:
They wanted a new method in the design documentation, in this case we implement the feature in a non breaking way.